pkg/flare/cluster_checks.go

```diff
@@ -22,7 +22,11 @@ import (
 
 // GetClusterChecks dumps the clustercheck dispatching state to the writer
 func GetClusterChecks(w io.Writer) error {
-	urlstr := fmt.Sprintf("https://localhost:%v/api/v1/clusterchecks", config.Datadog.GetInt("cluster_agent.cmd_port"))
+	ipcAddress, err := config.GetIPCAddress()
+	if err != nil {
+		return err
+	}
+	urlstr := fmt.Sprintf("https://%v:%v/api/v1/clusterchecks", ipcAddress, config.Datadog.GetInt("cluster_agent.cmd_port"))
```

Comment:
I think the address here is supposed to be the address of the IPC of the cluster agent, not of the node agent, see https://github.com/DataDog/datadog-agent/blob/4a6a8c9788ea868bdc88fb24a8694d1a7c829de3/cmd/cluster-agent/app/clusterchecks.go#L48.

So probably best to leave this to `localhost`.